src/index.html

```diff
@@ -1,13 +1,463 @@
+    <main>
+      <section class="page__section about-us" id="about-us">
+        <div class="container">
+          <div class="about-us__content grid">
+            <div class="about-us__name grid__item grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <p class="about-us__uptitle">
+                Plan your visit
+              </p>
+              <h2 class="about-us__title page__section-title">
+                Museum hours
+              </h2>
+            </div>
+            
+            <div class="about-us__info grid__item grid__item--tablet-4-6 grid__item--desktop-6-12">
+              <h3 class="about-us__info-title ">
+                The Met Fifth Avenue
+              </h3>
+
+              <div class="about-us__info-row">
+                <p class="about-us__working-hours">
+                  10 am - 5 pm
+                </p>
+                <p class="about-us__working-days">
+                  Sun, Mon, Tue, Thu
+                </p>
+              </div>
+
+              <div class="about-us__info-row">
+                <p class="about-us__working-hours">
+                  10 am - 9 pm
+                </p>
+                <p class="about-us__working-days">
+                  Fri, Sat
+                </p>
+              </div>
+
+              <div class="about-us__info-row">
+                <p class="about-us__working-hours">
+                  Closed
+                </p>
+                <p class="about-us__working-days">
+                  Wen
+                </p>
+              </div>
+
+              <div class="about-us__info-column">
+                <p class="about-us__working-hours">
+                  Closed on
+                </p>
+                <p class="about-us__working-days">
+                  Thanksgiving Day, Dec 25, Jan 1, and the first Monday in May.
+                </p>
+              </div>
+
+
+              <h3 class="about-us__info-title">
+                The Met Cloisters
+              </h3>
+
+              <div class="about-us__info-row">
+                <p class="about-us__working-hours">
+                  10 am – 4.30 pm
+                </p>
+                <p class="about-us__working-days">
+                  Thu - Tue
+                </p>
+              </div>
+
+              <div class="about-us__info-row">
+                <p class="about-us__working-hours">
+                  Closed
+                </p>
+                <p class="about-us__working-days">
+                  Wen
+                </p>
+              </div>
+
+              <div class="about-us__info-column">
+                <p class="about-us__working-hours">
+                  Closed on
+                </p>
+                <p class="about-us__working-days">
+                  Thanksgiving Day, Dec 25, Jan 1
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      
+      <section class="page__section on-view" id="now-on-view">
```

Comment:
https://prnt.sc/8XpbZSc27urb => just one small fix in this section, I guess it will be look better with z-index in hover effect